tests/app/authentication/test_authentication.py

```diff
@@ -278,8 +278,10 @@ def test_authentication_returns_error_when_service_doesnt_exit(
 
 def test_authentication_returns_error_when_service_inactive(client, sample_api_key):
     sample_api_key.service.active = False
-    token = create_jwt_token(secret=str(sample_api_key.id), client_id=str(sample_api_key.service_id))
+    # we now need to save the model because we'll read using read-db engine
+    dao_update_service(sample_api_key.service)
```

Comment:
I don't follow this logic. The test db points to the write db.

Reply:
It was an in-memory object that was not saved to db. Previous assumption must have been that the change to the property would've been picked up by the service on the other side of the request. Since we're not using the same service now, but reading in a new one from the  db using read-db engine, we wouldn't pick up the change unless it was saved to the database.